### PR TITLE
Add doc skeleton for diskful and diskless install

### PR DESCRIPTION
**Pull request by daniceexi**

### CODE REVIEW

**Comment by whowutwut** [comment]
@daniceexi  You have a .swp file that was added as part of the commit.

File: docs/source/guides/admin-guides/manage_clusters/common/diskful.rst [hunk 1]
```diff
@@ -1,2 +0,0 @@
-Diskful Installation
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/diskful.rst]: Is this file delete intentional?

File: docs/source/guides/admin-guides/manage_clusters/common/diskless.rst [hunk 2]
```diff
@@ -1,2 +0,0 @@
-Diskless Installation
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/diskless.rst]: Is this file delete intentional?

File: docs/source/guides/admin-guides/manage_clusters/x86_64/diskful/index.rst [hunk 3]
```diff
@@ -1,4 +1,3 @@
 .. ========================
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/x86_64/diskful/index.rst]: Does not point to any file, causing lots of WARNINGS in the build

```
None:None: WARNING: toctree contains reference to document u'guides/admin-guides/manage_clusters/x86_64/diskful/index' that doesn't have a title: no link will be generated
```

File: docs/source/guides/admin-guides/manage_clusters/x86_64/diskless/index.rst [hunk 4]
```diff
@@ -1,4 +1,3 @@
 .. ========================
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/x86_64/diskless/index.rst]: Does not point to any file, causing lots of WARNINGS in the build
```
None:None: WARNING: toctree contains reference to document u'guides/admin-guides/manage_clusters/x86_64/diskless/index' that doesn't have a title: no link will be generated
```

File: docs/source/guides/admin-guides/manage_clusters/common/diskful.rst (hunk 1 above)
Comment (daniceexi) [inline docs/source/guides/admin-guides/manage_clusters/common/diskful.rst]: yes

File: docs/source/guides/admin-guides/manage_clusters/common/diskless.rst (hunk 2 above)
Comment (daniceexi) [inline docs/source/guides/admin-guides/manage_clusters/common/diskless.rst]: yes

File: docs/source/guides/admin-guides/manage_clusters/x86_64/diskful/index.rst (hunk 3 above)
Comment (daniceexi) [inline docs/source/guides/admin-guides/manage_clusters/x86_64/diskful/index.rst]: Since we will clone from powerle for x86 later, just blank this file.

**Comment by daniceexi** [comment]
I did some change base our discussion. We can afford waiting more for the skeleton, so I just check in the code. If you have more comments, please send me here.

**Comment by daniceexi** [comment]
I meant CANNOT afford more waiting. Sorry :)

**Comment by whowutwut** [comment]
Ok with the merge, we should handle resolving some of the warnings that appear. Maybe after merge docs to xcat-core

**Comment by daniceexi** [comment]
No problem.

**Pull request merged**